libs/checkpoint-validation/README.md

```diff
@@ -0,0 +1,103 @@
+# @langchain/langgraph-checkpoint-validation
```

Comment:
We should add a section to the LangGraph docs around contributing! I can do that in a separate PR after we merge this.